profile/README.md

```diff
@@ -0,0 +1,10 @@
+# The conda Organization
+conda is an open source organization and the community behind [conda](https://github.com/conda/conda), [conda-incubator](https://github.com/conda-incubator), and [conda-archive](https://github.com/conda-archive).<br>
+
```

Comment:
I like the idea of a visual, but this one is a little confusing. Maybe it would be more clear as some sort of timeline that goes from left to right? (i.e. the life and death of a conda project 😂)

Reply:
The governance policy says about "Incorporate a Software Project into the main conda Organization":

> There is not [a] requirement for a project to incubate before a vote of this nature can be called.

Meaning that projects may also be directly added to the conda organization upon request to the steering council. The majority of projects would go through incubation though.

https://github.com/conda-incubator/governance#incorporate-a-software-project-into-the-main-conda-organization

Reply:
I intentionally didn't show all the possible paths as it got confusing. I originally had a left-to-right progression from community to conda-incubator to conda to conda-archive. Maybe we should switch back to the simple graph and include a note that the full details are outlined in governance?

Reply:
Done!